cmake/Sanitizer.cmake

```diff
@@ -0,0 +1,69 @@
+# Set appropriate compiler and linker flags for sanitizers.
+#
+# Usage of this module:
+#  enable_sanitizers("address;leak")
+
+# Add flags
+macro(enable_sanitizer sanitizer)
+  if(${sanitizer} MATCHES "address")
+    find_package(ASan REQUIRED)
+    set(SAN_COMPILE_FLAGS "${SAN_COMPILE_FLAGS} -fsanitize=address")
+    if (ASan_FOUND)
```

Comment:
I believe we can omit these `if (xxx_FOUND)`s due to recently added `REQUIRED` flag to `find_package` commands.